documentation/developer_updates.md

```diff
@@ -3,6 +3,50 @@
 Find out what the Bolt team is working on and why we're making the decisions
 we're making.
 
+## November 2020
+
+### Puppet 7 Release Prep
+
+[Puppet 7](https://puppet.com/docs/puppet/7.0/puppet_index.html) is set to be released November
+19th, 2020. While the Bolt package won't be pulling in Puppet 7 until early next year, the
+`puppet_agent::install` task bundled with Bolt will pick up Puppet 7 agents by default when the
+packages are available. This means that using the `apply_prep()` Bolt plan function or the install
+task directly will be pulling in a major version bump for the agent. While we don't anticipate any
+of the changes in the agent to affect Bolt users, if you find you need to use the Puppet 6 agent you
+can configure `apply_prep()` and the task to do so easily.
```

Comment:
```suggestion
task directly will be pulling in a major version bump for the agent. While we don't anticipate any
of the changes in the agent to affect Bolt users, if you find you need to use the Puppet 6 agent, you
can configure `apply_prep()` and the the `puppet_agent::install` task to do so.
```
We usually avoid words like "easily" and "quickly" as user mileage may vary ;)

Reply:
So smart and considerate!